src/providers/colors/ColorProvider.cpp

```diff
@@ -89,7 +88,7 @@ void ColorProvider::initTypeColorMap()
     {
         this->typeColorMap_.insert(
             {ColorType::Subscription,
-             std::make_shared<QColor>(backgrounds.subscription)});
+             std::make_shared<QColor>(HighlightPhrase::FALLBACK_COLOR)});
```

Comment:
This should have its own fallback color, to emulate the sub color. Right now, it would fall back to the red highlight color which doesn't seem right :astonished:

Reply:
Thanks, see 5fe20b3.